presto-main/pom.xml

```diff
@@ -499,6 +499,11 @@
             <artifactId>ratis-common</artifactId>
             <optional>true</optional>
         </dependency>
+        <dependency>
```

Comment:
@tdcmeehan 
Instead of using Mockito to create mock objects and define their behavior, can I manually write simple classes (stubs) that simulate the behavior of the actual classes?

Do you have any suggestion here in order to verify the codebase if value passed to request header is there correclty?

Reply:
Yes, that is how we go about this.

Reply:
@tdcmeehan 
I have refactored the code that was added for the test case.